agent/config/ank-agent.conf

```diff
@@ -0,0 +1,46 @@
+# ank-agent.conf is the configuration file for an Ankaios agent.
```

Comment:
I would not write the name of the file. It could well be that we add an option to the agent that specifies the exact path to the file including the name and a user could use a different name:
```suggestion
# This is the configuration file for an Ankaios agent.
```

Edit: 
The same goes for the other config files.